style.css

```diff
@@ -45,10 +45,11 @@ main {
 .tool_bar a {
   text-decoration: none;
   font-family: 'Roboto', sans-serif;
-  font-weight: 500;
-  font-size: 2rem;
+  font-weight: 700;
+  font-size: 32px;
   color: #ffff;
   cursor: pointer;
+  padding-left: 12px;
 }
```

Comment:
- [ ] Kindly follow the same design guidelines provided on Figma, the nav bar should look like this:
  ![image](https://user-images.githubusercontent.com/20567503/208525046-33dc2794-762c-4f3e-a3a2-5efe6e934c35.png)
  Yours right now looks like this:
  ![image](https://user-images.githubusercontent.com/20567503/208525121-8ace6e59-eb88-4f0f-8922-ea139326511a.png)
  Fixing the margins should be enough in my opinion, you just have a lot of margins on the sides (left & right)

Reply:
Hi, @mirouhml thank you for your review, it is great to have the opportunity to learn from you, however, I would like to ask you if it is mandatory to change my personal logo to "John Doe". Since this is my personal Portfolio Web I think it makes sense to add my personal logo instead of a random word, what do you think about this? , thank you so much!